src/background/Wallet/Wallet.ts

```diff
@@ -1796,6 +1850,13 @@ export class Wallet {
     await this.ensureTestnetModeForChainId(chainId, tabId);
   }
 
+  async getRpcUrlSolana() {
```

Comment:
Can we create something like `getRpcUrlByNetworkId` instead to make it less specific?